### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by joewhale**
* Updated Readme
* WIP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,19 @@
...
+# Up All Knight
+This application allows you to play a 1 vs 1 chess game online.
 
-* Services (job queues, cache servers, search engines, etc.)
+## Built with
+* [Ruby on Rails](http://rubyonrails.org/)
 
-* Deployment instructions
+## Authors
+* Joel Choi
```
Comment (lasley) [inline README.md]: Check with the team before adding there's, but IMO email belongs here too:
```
* A Name <a_name@derp.com>
```

Or maybe link to Github profile instead.

Comment (GonSanchezS) [inline README.md]: I would rather link to my twitter (@gonsanchezs) if possible 👍

Comment (kellylougheed) [inline README.md]: I'm good with github! Maybe we can all go in later and add a link of our choice.

Comment (joewhale) [inline README.md]: Okay, I made a comment in Trello, but we can discuss in our next meeting or just on Slack what links people want to add. I've left only the names in there for now.

Comment (PamBWillenz) [inline README.md]: I'm also good with github - thanks Joel @joewhale

**Pull request merged**